### PR TITLE
TV Android app - Add bottom navigation, Add screen for every bottom navigation.

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
- TV Android app has one screen, where everything UI related is merged.
- Customers are not able to use all the UI commands
- Developers are not able to extend the existing UI commands as there's no space 

#### Change overview
see above

#### Testing
Run `./scripts/run_in_build_env.sh \  
                  "./scripts/tests/run_test_suite.py \
                     --chip-tool ./out/debug/standalone/chip-tool \
                     run \
                     --iterations 1 \
                     --all-clusters-app ./out/debug/standalone/chip-all-clusters-app \
                     --tv-app ./out/debug/standalone/chip-tv-app \
                  "`

### CODE REVIEW

**Comment by lazarkov** [comment]
Screenshot 2022-05-23 at 01 09 19
Screenshot 2022-05-23 at 01 09 42
Screenshot 2022-05-23 at 01 09 53
Screenshot 2022-05-23 at 01 10 10

**Review by chrisdecenzo — APPROVED** [review]

File: examples/tv-app/android/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Android build folder
+android/App/app/build/
```
Comment (amitnj) [inline examples/tv-app/android/.gitignore:2]: There should not be anything in this folder anymore. Is there something here?

File: examples/tv-app/android/App/platform-app/src/main/res/values/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<resources>
```
Comment (amitnj) [inline examples/tv-app/android/App/platform-app/src/main/res/values/arrays.xml:1]: Where is this used?

File: examples/tv-app/android/.gitignore (hunk 1 above)
Comment (lazarkov) [reply examples/tv-app/android/.gitignore]: Updated. Added to a few lines to the App/.gitignore file.

File: examples/tv-app/android/App/platform-app/src/main/res/values/arrays.xml (hunk 2 above)
Comment (lazarkov) [reply examples/tv-app/android/App/platform-app/src/main/res/values/arrays.xml]: Removed.

**Review by amitnj — APPROVED** [review]

**Review by hawk248 — APPROVED** [review]

**Pull request merged**